src/cryptoadvance/specter/templates/wallet/history/components/tx_table.jinja

```diff
@@ -13,18 +15,39 @@
                 <th class="optional">Address</th>
                 <th>Amount</th>
                 <th>Status</th>
+		{% if specter.config.validate_merkle_proofs %}
+		  <th>
```

Comment:
```suggestion
		  <th class="optional">
```

Just need this not to break mobile view

Reply:
Are you sure, the amount/status aren't marked `class="optional"`? This whole section only appears if you have `specter.config.validate_merkle_proofs`.

Reply:
Yes I'm sure, the amounts and status are small enough to fit in mobile display, but the block header is too large... you can include it in the mobile-only pop up which has the txid and all, but don't have to if you find it confusing to implement.

Reply:
Got it, that makes sense. Thanks for the detailed explanation, I've barely used jinja before!

Reply:
Separately, maybe the class should be renamed to something like `mobile-compatible`, `collapsible`, `squeezable`, etc in the future? Optional expresses something a little different (that the value is not required), but there are formatting benefits when it's guaranteed.

Reply:
Yeah that definitely make sense, I'm not great with naming lol